react/features/base/i18n/i18next.ts

```diff
@@ -35,22 +37,18 @@ const COUNTRIES = _.merge({}, COUNTRIES_RESOURCES, COUNTRIES_RESOURCES_OVERRIDES
  */
 export const LANGUAGES: Array<string> = Object.keys(LANGUAGES_RESOURCES);
 
-/**
- * The languages for the top section of the translation language list.
- *
- * @public
- * @type {Array<string>}
- */
-export const TRANSLATION_LANGUAGES_HEAD: Array<string> = [ 'en' ];
+export const i18n = {
+    get translationLanguages() {
+        const config: IConfig = APP.store.getState()['features/base/config'];
```

Comment:
The APP global does not exist on mobile. What's your plan there?

Reply:
I could just use the defaults if's not defined(but then we can't overwrite the languages). I'm not sure what how could i do this on mobile. But if you can help with this i'm open to it.